pytest_pyodide/config.py

```diff
@@ -53,7 +53,7 @@ def get_initialize_script(self) -> str:
         return self.initialize_script
 
     def add_node_extra_globals(self, l: Iterable[str]):
-        self.node_extra_globals.update(l)
+        self.node_extra_globals.extend(l)
```

Comment:
I wonder if there is some problem with our mypy config, since mypy should have caught this. cc @ryanking13 @agriyakhetarpal